gourmet/tests/test_foodnetwork_plugin.py

```diff
@@ -49,20 +56,20 @@ def test_parse(self):
         ingredients = [r[0] for r in result if r[1] == "ingredients"]
         name = [r for r in result if r[1] == "title"][0][0]
         instructions = [r[0] for r in result if r[1] == "recipe"]
+        preptime = [r for r in result if r[1] == "preptime"][0][0]
+        cooktime = [r for r in result if r[1] == "cooktime"][0][0]
+        yields = [r for r in result if r[1] == "yields"][0][0]
 
         # Check results
-        self.assertEqual(len(ingredients), 8)
-
-        self.assertTrue('Pan-Roasted Chicken with Oranges and Rosemary' in name)
-
-        self.assertTrue('Heat oven to 450 degrees F and arrange rack in middle.' in instructions)
-
-        # The text to check for is part of a list element, not the full list element.
-        # Therefore we are creating a single string with the instructions for this check.
-        self.assertTrue('Let rest 5 minutes before serving.' in "\n".join(instructions))
+        self.assertEqual(len(ingredients), 11)
+        self.assertEqual(preptime, "15 mins")
+        self.assertTrue(cooktime, "25 mins")
+        self.assertEqual(yields, "4")
 
-        self.assertFalse('You must be logged in to review this recipe.' in instructions)
+        self.assertEqual(name, "Pan Roasted Chicken Thighs with Grapes and Olives")
 
+        self.assertTrue(
+            "Place the skillet underneath the broiler to crisp the chicken skin, about 2 minutes. Watch carefully to avoid burning." in instructions)
```

Comment:
Please replace with https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn for cooler error messages